gym_solo/core/termination.py

```diff
@@ -0,0 +1,48 @@
+from abc import ABC, abstractmethod
+
+
+class Termination(ABC):
+    @abstractmethod
+    def __init__(self):
+        """This is used to define the termination condition
+        """
+        pass
+    
+    @abstractmethod
+    def is_terminated(self):
```

Comment:
nit: add type hint, this returns a bool, right?

Reply:
Yes and Done